_docs/quickstarts/forseti-security/index.md

```diff
@@ -4,30 +4,58 @@ order: 002
 ---
 # {{ page.title }}
 
-This quickstart explains how to use the Forseti setup wizard to get started
-quickly with Forseti Security for Google Cloud Platform (GCP) projects.
+This quickstart explains how to use the Forseti setup wizard, which helps to 
+automate some of the Forseti installation and setup process.
+
+Setup wizard is not available prior to version 1.1.0.
 
 ## Before you begin
 
-To complete this quickstart, you will need:
+Prior to running the setup wizard, you will need:
 
   - A GCP organization for which you want to deploy Forseti.
   - Org Admin IAM role in order for the script to assign the Forseti 
   service account roles on the organization IAM policy.
+  - A GCP project dedicated to Forseti.
+  - Enable billing on the project.
+
 
 ## Setting up Forseti Security
 
-The setup wizard presents a series of prompts to guide you through setting
-up a Forseti deployment, then generates a deployment template and configuration
-file based on your input.
+The setup wizard automatically determines setup information, generates a 
+deployment template, and creates a Forseti deployment.
+
+We recommend using Cloud Shell to run Forseti setup wizard to ensure 
+that you have the latest version of gcloud and the necessary gcloud components.
```

Comment:
I assume that "gcloud" refers to the gcloud command-line tool? All instances of "gcloud" should be updated for branding: please use either "Cloud SDK" or "gcloud command-line tool".

Also, this paragraph adds info that I don't really "need" to complete the quickstart. It's probably best to remove it in favour of a bullet in "Before you begin".